src/main/webapp/app/exercises/shared/exercise/exercise.utils.ts

```diff
@@ -132,24 +132,24 @@ export const getExerciseDueDate = (exercise: Exercise, participation?: Participa
     }
 };
 
-/**
- * Checks if the given exercise has student participations.
- *
- * @param exercise
- * @return {boolean}
- */
-export const hasStudentParticipations = (exercise: Exercise) => {
-    return exercise.studentParticipations && exercise.studentParticipations.length > 0;
-};
-
 /**
  * Determines if the exercise can be started, this is the case if:
- * - It is after the start date or the participant is at least a tutor
```

Comment:
This comment is probably still valiid?

Reply:
No, this function does not check this. This is only enforced in the html by disabling the buttons